scripts/SMCodeGen/dataModels/ctxtManagerAppModel.json

```diff
@@ -99,7 +99,11 @@
 							{
 								"Name": "pdnAddr",
 								"Type": "Paa"
-							}
+							},
+							{
+                                                                "Name": "sgwCtrlPIP",
```

Comment:
What's this P stats for in sgwCtrl**P**IP ?

Reply:
changed the name sgwCtrlPIP to sgwCtrlIP. uint64_t  is there because of the IP variable in in_addr structure
struct in_addr {
    unsigned long s_addr;  // load with inet_aton()
};